core/testcontainers/core/container.py

```diff
@@ -22,15 +34,37 @@ class DockerContainer:
         >>> with DockerContainer("hello-world") as container:
         ...    delay = wait_for_logs(container, "Hello from Docker!")
     """
-    def __init__(self, image: str, docker_client_kw: Optional[dict] = None, **kwargs) -> None:
-        self.env = {}
-        self.ports = {}
-        self.volumes = {}
+    def __init__(
```

Comment:
Can we do the same with `network` and `network_aliases`?  I suspect we need to rebase this PR in order to pickup these attributes.